src/azure-cli/azure/cli/command_modules/backup/custom_common.py

```diff
@@ -282,7 +280,7 @@ def _get_containers(client, backup_management_type, status, resource_group_name,
     containers = custom_help.get_list_from_paged_response(paged_containers)
 
     if container_name and custom_help.is_native_name(container_name):
-        return [container for container in containers if container.name == container_name]
+        return [container for container in containers if container.name.lower() == container_name.lower()]
```

Comment:
I want to confirm with you that `container.name` and `container_name` are not `None` at any time, right?

Reply:
we are checking the `container_name` is not `None` in the `if` clause. And `container.name` should not be `None` as it is a mandatory parameter coming from the service.

Reply:
Got it, thanks~